src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -33,6 +33,56 @@
 """ Space needed to reach the root of the literal block. """
 BASE_INDENT = " " * 12
 
+CATEGORIES_USED_ON_DESCRIPTION = [
+    "time",
+    "length",
+    "temperature",
+    "volume flow rate",
+    "volume fraction",
+    "mass flow rate",
+    "standard volume per standard volume",
+    "pressure",
+    "mass fraction",
+    "mole per mole",
+    "molar thermodynamic energy",
+    "angle per time",
+    "dimensionless",
+    "flow coefficient",
+    "standard volume per time",
+    "productivity index",
+    "power",
+    "velocity",
+    "heat transfer coefficient",
+    "density",
+    "thermal conductivity",
+    "specific heat capacity",
+    "emissivity",
+    "volumetric thermal expansion",
+    "dynamic viscosity",
+    "standard volume per standard volume",
+    "mass per mol",
+    "molar volume",
+]
+
+
+def generate_list_of_units(category: str) -> str:
+    """ Return an admonition with toggle to show the units available for a given category. """
+    from barril.units import UnitDatabase
+
+    unit_database = UnitDatabase.GetSingleton()
+    title = unit_database.GetCategoryInfo(category).caption.title()
+    units = unit_database.GetCategoryInfo(category).valid_units_set
+    info = unit_database.unit_to_unit_info
+    body = [f"    :{unit}: {info[unit].name}" for unit in units]
+    lines = [
+        f".. admonition:: Available units for category '{title}'",
```

Comment:
While I understand the reasoning, I don't think we should use `title` here: the user is supposed to write `"time"`, not `"Time"`, so using `'Time'` here is confusing.

Reply:
I noticed you changed the code, but the docs are still showing the category as a title:

![image](https://user-images.githubusercontent.com/1085180/101623527-5757a880-39f7-11eb-938d-7bc4215471c2.png)

Did you forget to run the script that generates the docs?

Reply:
Exactly, I forgot to run `inv cog` and the CI was not catching changes on the `alfacase_definitions` folder
Commit 221c7d4 adds this check and commit 6aa24a7 update the list ;)